#Details of service data types
float64 changeThreshold
---
bool response
